django_filters/filters.py

```diff
@@ -20,13 +20,28 @@
 
 
 __all__ = [
-    'Filter', 'CharFilter', 'BooleanFilter', 'ChoiceFilter',
-    'TypedChoiceFilter', 'MultipleChoiceFilter', 'DateFilter',
-    'DateTimeFilter', 'IsoDateTimeFilter', 'TimeFilter', 'ModelChoiceFilter',
-    'ModelMultipleChoiceFilter', 'NumberFilter', 'NumericRangeFilter', 'RangeFilter',
-    'DateRangeFilter', 'DateFromToRangeFilter', 'DateTimeFromToRangeFilter',
-    'TimeRangeFilter', 'AllValuesFilter', 'MethodFilter', 'UUIDFilter',
-    'DurationFilter',
+    'AllValuesFilter',
+    'BooleanFilter',
+    'CharFilter',
+    'ChoiceFilter',
+    'DateFilter',
+    'DateFromToRangeFilter',
+    'DateRangeFilter',
+    'DateTimeFilter',
+    'DateTimeFromToRangeFilter',
```

Comment:
Thanks, it looks like it was another one that wasn't included in `__all__` to begin with.